sdks/python/setup.py

```diff
@@ -269,22 +269,21 @@ def get_portability_package_data():
             # Transitive dep. Required for google-cloud-spanner v1.
             # See: https://github.com/apache/beam/issues/22454
             'google-api-core!=2.8.2,<3',
-            # NOTE: Maintainers, please do not require google-auth>=2.x.x
-            # Until this issue is closed
-            # https://github.com/googleapis/google-cloud-python/issues/10566
             'google-auth>=1.18.0,<3',
             'google-auth-httplib2>=0.1.0,<0.2.0',
             'google-cloud-datastore>=1.8.0,<2',
             'google-cloud-pubsub>=2.1.0,<3',
             'google-cloud-pubsublite>=1.2.0,<2',
-            # GCP packages required by tests
```

Comment:
I don't think this line is accurate. Below packages are used by IOs.

Reply:
So none of them are actually test dependencies? Should they all be merged back into gcp extra?

If yes can do although value of pr is low and maybe better to just close if spanner/bigquery upper bound are still required.

Reply:
I did some code archaeology here to see where this comment came from. It dates back to when we added a BQ E2E test in 2017: https://github.com/apache/beam/commit/dd32c26622ef9d0aa9e8d0c3863ac6660ed336b7#diff-1275c48808de339ef6f282d844c83ec441b5cfa0debc373fdcb7dba497da4fc8

"required by tests" seems to be a distinction that wasn't well thought out and hasn't been consistently applied, I think we should just remove this comment.